### ISSUE TITLE
Vertex z dependent reconstruction efficiency

### ISSUE THREAD

**Issue by aaust** [JeffersonLab/halld_recon#166]
I generated pi+ pi- p events for t in [0,0.1] GeV2. For the events that make it through the analysis, the protons have up to 300MeV and are emitted at 73 degrees:
 (black thrown, red thrown for reconstructed, green reconstructed)

![proton_theta](https://user-images.githubusercontent.com/8807112/58274714-6c131380-7d61-11e9-9403-441aee000069.png)
![proton_p](https://user-images.githubusercontent.com/8807112/58274715-6c131380-7d61-11e9-800c-7c0b518882d0.png)

For these events, the z vertex distribution shows a hump between 75 and 80cm:
![vertex_z](https://user-images.githubusercontent.com/8807112/58274125-31f54200-7d60-11e9-8ab8-027a84883e05.png)

As this happens for both Geant3 and Geant4, I did not believe this is specific to the simulation part. As a matter of fact, I also observe a strange vertex z distribution for real data at low t (between 0.05 and 0.10):
![vertexz_data](https://user-images.githubusercontent.com/8807112/58276122-a631e480-7d64-11e9-8826-c2446c4cbe9b.png)

**Comment by rjones30** [comment]
Hello Alex,

Why do the red and green curves not have the same integral, if both are for
the same set of reconstructed protons?

-Richard

**Comment by aaust** [comment]
@rjones30: the plugin monitoring_hists requires a minimal PID FOM of 5.73303E-7 to assign particle types to the tracks. This cut is applied to the green histograms for monitoring purposes.

**Comment by rjones30** [comment]
Alex,

Is there any reason to suppose that the red histogram is representative of
what the green histogram should show, given that the same cut was not
applied to them? Or is the green not to be compared to the red? -Richard

**Comment by aaust** [comment]
@rjones30: Yes, you can ignore the green histograms for now. The ratio between red and black gives you a crude reconstruction efficiency, that does not take resolution effects into account.
